website/docs/team.md

```diff
@@ -5,26 +5,61 @@ title: Team
 
 ## Meet the Docusaurus Team
 
-- Alexey Pyltsyn
-- Fanny Batista Vieira
-- Joel Marcey
-- Yangshun Tay
+### Alexey Pyltsyn
+
+<br/>
+
+![](https://github.com/lex111.png)  
+
+[@lex111 on Github](https://github.com/lex111)
+
+### Fanny Vieira
+
+<br/>
+
+![](https://github.com/fanny.png)   
+[@fanny on Github](https://github.com/fanny) · [@fannyvieiira on Twitter](https://twitter.com/fannyvieiira)
+
+### Joel Marcey  
+
+<br/>
+
+![](https://github.com/joelmarcey.png)  
+[@joelmarcey on Github](https://github.com/joelmarcey) · [@joelmarcey on Twitter](https://twitter.com/joelmarcey)
+
+### Yangshun Tay
+
+<br/>
+
+![](https://github.com/yangshun.png)  
+[@yangshun on Github](https://github.com/yangshun) · [@yangshunz on Twitter](https://twitter.com/yangshunz)
 
 ## Honorary Alumni
 
-- Endilie Yacop Sucipto
-- Wei Gao
+### Endilie Yacop Sucipto
+
+<br/>
+
+![](https://github.com/endiliey.png?width=75&height=75)  
+[@endiliey on Github](https://github.com/endiliey) · [@endiliey on Twitter ](https://twitter.com/endiliey)
+
+### Wei Gao
+
+<br/>
+
+![](https://avatars3.githubusercontent.com/u/2055384?s=460&u=09248b65169cb9909246dfa39b796deffc7bd251&v=4)  
+[@wgao19 on Github](https://github.com/wgao19) · [@wgao19 on Twitter](https://twitter.com/wgao19)
 
 ## Acknowledgements
 
 Docusaurus was originally created by Joel Marcey. Today, Docusaurus has a few hundred open source contributors. We’d like to recognize a few people who have made significant contributions to Docusaurus and its documentation in the past and have helped maintain them over the years:
 
-- Amy Lam
-- Cheng Lou
-- Christine Abernathy
-- Christopher Chedeau
+- [Amy Lam](https://github.com/amyrlam)
+- [Cheng Lou](https://github.com/chenglou)
+- [Christine Abernathy](https://github.com/caabernathy)
+- [Christopher Chedeau](https://github.com/vjeux)
 - Elvis Wolcott
```

Comment:
Most of these users are among the [top contributors to the repo](https://github.com/facebook/docusaurus/graphs/contributors)